### PR TITLE
Remove and replace duplicates in Animals and Adjectives:

### PR DESCRIPTION
**Pull request by huths0lo**
I will be using a port of Angry Purple Tiger for another project.  I have removed and replaced duplicate words, so that all are unique.  Would be ideal for the master version of Angry Purple Tiger to be consistent.

Animals removed: raccoon
Animals added: axoltl

Adjectives removed: skinny, short, long, short, cool, hot, tangy
Adjectives added: serene, ethereal, nebulous, quixotic, luminous, clandestine, effervescent

### CODE REVIEW

**Review by madninja — COMMENTED** [review]
Unfortunately that would change the names of all existing hotspots. Which would cause mayhem

**Comment by huths0lo** [comment]
> Unfortunately that would change the names of all existing hotspots. Which would cause mayhem

How would this cause mayhem?  The names make it easy for people to find them.  But thats now how they're addressed on net; so its just an aesthetic change for hotspot owners.

My very first hotspot "Brisk Mauve Bird" has a couple of duplicate names for existing hotspot.  So the names arent even unique.  The original code for angry purple tiger very much has duplicate names in; which although wouldnt solve all cases, it would at least provide better downstream use for things like what I'm building.

Basically not introducing those fixes, guarantees no one else will ever be able to use your code, without making changes.  I'm proposing making a one time fix, so that future code use can have identical results.  You could even put this in a separate branch that is the recommended branch to fork from.

**Comment by madninja** [comment]
> > Unfortunately that would change the names of all existing hotspots. Which would cause mayhem
> 
> How would this cause mayhem? The names make it easy for people to find them. But thats now how they're addressed on net; so it's just an aesthetic change for hotspot owners.

I know, you know but that's not how most hotspot owners see it.. Having their hotspot name change drastically is not expected. We 
> 
> My very first hotspot "Brisk Mauve Bird" has a couple of duplicate names for existing hotspot. So the names arent even unique. The original code for angry purple tiger very much has duplicate names in; which although wouldnt solve all cases, it would at least provide better downstream use for things like what I'm building.

Names will never be unique
> 
> Basically not introducing those fixes, guarantees no one else will ever be able to use your code, without making changes. I'm proposing making a one time fix, so that future code use can have identical results. You could even put this in a separate branch that is the recommended branch to fork from.

I understand your need.. I also don't like the duplicates but we can't just change the names without some community approval here.

If you want something different the alternative is to add a a feature that takes the wordlist identifier to use so code can pick up the wordlist they want. That way it can support multiple wordlists.

**Comment by huths0lo** [comment]
I get your point.  I havent done the math, but the percentage of hotspots that would become renamed would be pretty minimal.  Either way, it sounds like its a non starter.  But please do consider it for an alternative branch.  As I said; if there is one master source, it makes it so downstream builds can derive the same name.

I created a Python version of Angry Purple Tiger, which includes the original list: https://github.com/TheMorphium/pymorphic-purple-tiger

Yesterday I made a new application with a fourth word, and with no duplicates within each list (although theres a handful of dupes between the lists, because the words work in multiple categories.  I may take it a step further, and eliminate those as well).  Thats available here: https://pypi.org/project/PypeDreams/

You have a great tool.  I'd love to see it expanded on.
